jarviscli/christmas timer.py

```diff
@@ -0,0 +1,47 @@
+from plugin import plugin, pap35000
```

Comment:
Why are you importing ```pap35000``` and what is this supposed to do?

Also, file needs to be placed in jarviscli/pluings to be loaded.

Reply:
how I place it to jarviscli/pluings